piq/ssim.py

```diff
@@ -207,11 +211,14 @@ def multi_scale_ssim(x: torch.Tensor, y: torch.Tensor, kernel_size: int = 11, ke
            https://ece.uwaterloo.ca/~z70wang/publications/ssim.pdf,
            :DOI:`10.1109/TIP.2003.819861`
     """
-    _validate_input(input_tensors=(x, y), allow_5d=True, kernel_size=kernel_size, scale_weights=scale_weights)
+    _validate_input(
+        input_tensors=(x, y), allow_5d=True, kernel_size=kernel_size,
+        scale_weights=scale_weights, data_range=data_range
+    )
     x, y = _adjust_dimensions(input_tensors=(x, y))
-    if isinstance(x, torch.ByteTensor) or isinstance(y, torch.ByteTensor):
-        x = x.type(torch.float32)
-        y = y.type(torch.float32)
+
+    x = x / float(data_range)
+    y = y / float(data_range)
```

Comment:
In lines 54-58 of the same file you first convert tensors to `torch.float32` and then devide by `float(data_range)`. Here the first step is skipped. Please make these two cases consistent.